gammapy/data/tests/test_event_list.py

```diff
@@ -82,6 +83,12 @@ def test_write(self):
             dummy_events.table.meta["HDUCLAS1"] = "events"
             dummy_events.write("test.fits", overwrite=True)
 
+    def test_write_checksum(self):
+        self.events.write("test.fits", overwrite=True, checksum=True)
+        hdu = fits.open("test.fits")["EVENTS"]
+        assert "CHECKSUM" in hdu.header
```

Comment:
Can we test also the values of the CHECKSUM and DATASUM? One should be aware when any change will happen

Reply:
I'd rather test that when reading after writing everything is fine. Testing the exact checksum/datasum will break if anything changes in the file content without tolerance.